test/integration/component/test_escalations_vmware.py

```diff
@@ -331,3 +334,46 @@ def test3_attach_ISO_in_RHEL7OSVM(self):
         self.assertEqual(attachedIsoName, "vmware-tools.iso", "vmware-tools.iso not attached")
         return
 
+    @attr(tags=["advanced", "basic"], required_hardware="true")
+    def test_04_check_vm_snapshot_creation_after_Instance_creation(self):
+        """
+        @summary: Test  if Snapshot creation is successful
+        in the first 10 minutes of VM deployment
+
+        Step1: Create a VM with any Service offering
+        Step2: Create a VM snapshot
+        Step3: Verify if SS creation is successful within first
+               10 minutes of VM deployment
+        """
+
+        if self.hypervisor.lower() not in ['vmware']:
+            self.skipTest("This test case is only for vmware. Hence, skipping the test")
+        vm = VirtualMachine.create(
+            self.userapiclient,
+            self.services["virtual_machine"],
+            accountid=self.account.name,
+            domainid=self.account.domainid,
+            serviceofferingid=self.service_offering.id,
+            templateid=self.template.id,
+            zoneid=self.zone.id
+        )
+
+        start_time = time.time()
+
+        snapshot_created_1 = VmSnapshot.create(
+            self.userapiclient,
+            vm.id
+        )
+        self.assertIsNotNone(
+            snapshot_created_1,
+            "Snapshot creation failed"
+        )
+
```

Comment:
If create VM snapshot fails immediately after instance creation, the issue might exists.